kpi/paginators.py

```diff
@@ -142,3 +142,10 @@ class TinyPaginated(PageNumberPagination):
     Same as Paginated with a small page size
     """
     page_size = 50
+
+
+class OrganizationPagination(PageNumberPagination):
+    """
+    Pagination class for Organization
+    """
+    page_size = 10
```

Comment:
Is `10` a requirement?

Reply:
10 is not a requirement defined in the task. I need to check with you or the team to determine the appropriate value for it.

Reply:
@rajpatel24 Can we change this to limit offset pagination? Our table components on the frontend currently assume limit offset pagination and @noliveleger said it shouldn't cause any problems to use that here.

Reply:
@jamesrkiger I have removed the custom pagination class for the organization members API. It will now use `LimitOffsetPagination` by default.